recipe/meta.yaml

```diff
@@ -33,28 +33,27 @@ requirements:
     - entrypoints >=0.3
     - jinja2 >=3
     - jsonschema >=3.2.0,<4.0
```

Comment:
Several adjustments were made to dependencies and caps in https://github.com/elyra-ai/elyra/pull/3076, so we should apply them here.